tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js

```diff
@@ -0,0 +1,145 @@
+    const carbonPassword = 'admin';
+    const appName = 'App_A1';
+    const appDescription = 'Testing sharing app ';
+    const groupId = 'org1';
+    const apiName = `anonymous${Math.floor(Math.random() * (100000 - 1 + 1) + 1)}`;
+    const apiVersion = '2.0.0';
+    const apiContext = `anonymous${Math.floor(Math.random() * (100000 - 1 + 1) + 1)}`;
+
+
+    before(function () {
+        cy.carbonLogin(carbonUsername, carbonPassword);
+        cy.addNewUser(publisher, ['Internal/publisher', 'Internal/creator', 'Internal/everyone'], password);
+        cy.loginToPublisher(publisher, password);
+    
+    })
+
+    it("Test Application Sharing", () => {
+
+        cy.createAndPublishAPIByRestAPIDesign(apiName, apiVersion, apiContext);
+        cy.location('pathname').then((pathName) => {
+            const pathSegments = pathName.split('/');
+            const uuid = pathSegments[pathSegments.length - 2];
+            cy.logoutFromPublisher();
+
+            //Create Users in Devportal
+            cy.visit(`${Utils.getAppOrigin()}/devportal/apis`);
+            cy.get('#itest-devportal-sign-in').click();
+            cy.get('#registerLink').click();
+
+            //Creating user 1
+            cy.get('#username').type(user1);
+            cy.get('#registrationSubmit').click();
+
+            // There is an error and we have to fix it from IS side
+            Cypress.on('uncaught:exception', (err, runnable) => {
+                // returning false here prevents Cypress from failing the test
+                return false
+            });
+            cy.get('input[name="http://wso2.org/claims/givenname"]').type(user1);
+            cy.get('input[name="http://wso2.org/claims/lastname"]').type('test');
+            cy.get('#password').type(password);
+            cy.get('#password2').type(password);
+            cy.get('input[name="http://wso2.org/claims/emailaddress"]').type('user1@test.com');
+            cy.get('input[name="http://wso2.org/claims/organization"]').type('org1');
+            cy.get('#termsCheckbox').check().should('be.checked');
+            cy.get('#registrationSubmit').click();
+            cy.contains('Close').click() 
+
+            //Creating user 2
+            cy.get('#registerLink').click();
+            cy.get('#username').type(user2);
+            cy.get('#registrationSubmit').click();
+            cy.get('input[name="http://wso2.org/claims/givenname"]').type('user2');
+            cy.get('input[name="http://wso2.org/claims/lastname"]').type('test');
+            cy.get('#password').type(password);
+            cy.get('#password2').type(password);
+            cy.get('input[name="http://wso2.org/claims/emailaddress"]').type('user2@test.com');
+            cy.get('input[name="http://wso2.org/claims/organization"]').type('org1');
+            cy.get('#termsCheckbox').check().should('be.checked');
+            cy.get('#registrationSubmit').click();
+            cy.contains('Close').click();
+
```

Comment:
Shall we remove the extra line here?

Reply:
done

Reply:
Resolved with commit dc1fe3cdd1aec56b4bc0534300d8abc77ce18e58